data/mods/Xedra_Evolved/regional_overlay.json

```diff
@@ -23,13 +23,21 @@
           "mx_mayhem": 500,
           "mx_casings": 1000,
           "mx_corpses": 300,
-          "mx_prison_bus": 150,
-          "mx_trees_arvore": 50,
-          "mx_trees_arvore_salamander": 50,
-          "mx_pond_forest_arvore": 25,
-          "mx_pond_forest_undine": 25,
-          "mx_pond_forest_arvore2": 25,
-          "mx_pond_forest_undine2": 25
+          "mx_prison_bus": 150
+        }
+      },
+      "field": {
+        "chance": 75,
+        "extras": {
+          "mx_trees_map_salamander": 75,
+          "mx_trees_map_sylph": 75,
+          "mx_trees_map_ierde": 75,
+          "mx_forest_pond_homullus": 75,
```

Comment:
Any chance you could do the homullus over the one tile homeless encampment instead of a pond?  They aren't really thematically pond people.

Reply:
this should do the trick